docs/visual-testing/integrations/python-robot-framework.md

```diff
@@ -83,9 +83,12 @@ By default, only the current viewport is capture when taking a screenshot. Full
 You can customize the behavior by using the following options:
 
 | Property                  | Default | Example                                                           | Description                                                                                                                                                            |
-|---------------------------|---------|-------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `hide_after_first_scroll` | `[]`    | `['.your-class-name','#an-id-example','div.more-specific-class']` | One or more CSS selectors selectors that we should remove from the page after the first scroll. Useful for hiding fixed elements such as headers, cookie banners, etc. |
+|:--------------------------|:--------|:------------------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `delay_after_scroll_ms`   | `0`     | `300`                                                             | Delay in ms after scrolling and before taking screenshots. The default value is 0. We recommend using this option for lazy loading content.                            |
+| `disable_css_animation`   | `true`  | `false`                                                           | Disable CSS animations and the input caret in the app. The default value is true.                                                                                      |
+| `hide_after_first_scroll` | `[]`    | `['.your-class-name','#an-id-example','div.more-specific-class']` | One or more CSS selectors that we should remove from the page after the first scroll. Useful for hiding fixed elements such as headers, cookie banners, etc. |
+| `hideScrollBars`          | `true`  | `false`                                                           | Hide all scrollbars in the app. The default value is true.                                                                                                             |
```

Comment:
Atleast in Robot, these are snake case: https://github.com/saucelabs/visual-sdks/blob/0384c59f5a3cb415136fd4bd301feaba2c0cf1ee/visual-python/src/saucelabs_visual/frameworks/robot.py#L83 (see the parsed value that we're `.get`ting from

Reply:
@omacranger I fix it. Do you have any other comments?